src/Microsoft.PowerShell.Commands.Utility/resources/TestJsonCmdletStrings.resx

```diff
@@ -124,7 +124,7 @@
     <value>Cannot parse the JSON.</value>
   </data>
   <data name="InvalidJsonAgainstSchemaDetailed" xml:space="preserve">
-    <value>The JSON is not valid with the schema: {0} at {1}</value>
```

Comment:
When I updated, I tested this locally with a schema & instance that had a problem at the root.  The error that's output just ends with "at", and the fact that there's an empty JSON Pointer indicating the root object isn't conveyed.  I think the quotes help.